storage-engine/titan-configuration.md

```diff
@@ -110,19 +110,38 @@ To adjust Titan-related parameters using TiUP, refer to [Modify the configuratio
     rate-bytes-per-sec = 0
     ```
 
-## Disable Titan (experimental)
+## Disable Titan
 
 To disable Titan, you can configure the `rocksdb.defaultcf.titan.blob-run-mode` option. The optional values for `blob-run-mode` are as follows:
 
 - When the option is set to `normal`, Titan performs read and write operations normally.
 - When the option is set to `read-only`, all newly written values are written into RocksDB, regardless of the value size.
 - When the option is set to `fallback`, all newly written values are written into RocksDB, regardless of the value size. Also, all compacted values stored in the Titan blob file are automatically moved back to RocksDB.
 
-To disable Titan, set `blob-run-mode = "fallback"` and perform a full compaction using tikv-ctl. After that, check the monitoring metrics, confirm that the blob file size decreases to `0`. Then you can set `rocksdb.titan.enabled` to `false` and restart TiKV.
+To fully disable Titan for all existing and future data, you can follow these steps:
 
-> **Warning:**
->
-> Disabling Titan is an experimental feature. It is **NOT** recommended to use it if not necessary.
++ First update the configuration of the TiKV nodes you wish to disable Titan. This can be done either with `tiup cluster edit-config` followed by `tiup cluster reload`, or manually updating the configuration file and restarting TiKV.
+
+    ```toml
+    [rocksdb.defaultcf.titan]
+    blob-run-mode = "fallback"
+    discardable-ratio = 1.0
+    ```
+
++ Then perform a full compaction using tikv-ctl. This process will consume large amount of I/O and CPU resources.
+
+    ```bash
+    tikv-ctl --pd <PD_ADDR> compact-cluster --bottommost force
+    ```
+
++ After the compaction is finished, you should wait for the "Blob file count" metrics under "TiKV-Details/Titan - kv" to decrease to `0`.
```

Comment:
In [this document](https://docs.pingcap.com/tidb/stable/grafana-tikv-dashboard#titan---all), the name of the monitoring panel is "Titan - All". Can you confirm which is correct? @tabokie

Reply:
It's dynamically named. When titan is enabled, it will be "Titan - kv".

Reply:
Got it. Thanks.